helmfile.d/traefik.yaml

```diff
@@ -0,0 +1,20 @@
+repositories:
+  - name: traefik
+    url: 
```

Comment:
There's no url for the traefik repo, im not sure what it should be but it needs to be something